### PR TITLE
Move a bunch of IIIF-related pieces out of the common lib and into catalogue

### PR DESCRIPTION
**Pull request by alexwlchan**
They're only used there so they aren't really "common". Will this have any effect on bundle size or build time? Let's find out…

### CODE REVIEW

File: catalogue/webapp/components/ArchiveBreadcrumb/ArchiveBreadcrumb.tsx [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import { Work } from '@weco/common/model/catalogue';
 import styled from 'styled-components';
-import DropdownButton from '../DropdownButton/DropdownButton';
-import Icon from '../Icon/Icon';
+import DropdownButton from '@weco/common/views/components/DropdownButton/DropdownButton';
+import Icon from '@weco/common/views/components/Icon/Icon';
 import WorkTitle from '@weco/common/views/components/WorkTitle/WorkTitle';
```
Comment (jamesgorrie) [inline catalogue/webapp/components/ArchiveBreadcrumb/ArchiveBreadcrumb.tsx:5]: Could we move this one too?

**Review by jamesgorrie — APPROVED** [review]
This work is really nice to see happening!

File: catalogue/webapp/components/ArchiveBreadcrumb/ArchiveBreadcrumb.tsx (hunk 1 above)
Comment (alexwlchan) [reply catalogue/webapp/components/ArchiveBreadcrumb/ArchiveBreadcrumb.tsx:5]: Yeah, but I'll do that separately. This bit works, let's shake out some more stuff now.

**Comment by alexwlchan** [comment]
Attaching to https://github.com/wellcomecollection/wellcomecollection.org/issues/6927

**Pull request merged**